test/test_viscous.py

```diff
@@ -240,6 +240,45 @@ def test_diffusive_heat_flux(actx_factory):
         assert discr.norm(j[ispec+1] - exact_j, np.inf) < tol
 
 
+@pytest.mark.parametrize("dim", [1, 2, 3])
+def test_local_max_species_diffusivity(actx_factory, dim):
+    actx = actx_factory()
+    nel_1d = 5
```

Comment:
```suggestion
    nel_1d = 4
```
(The `n` parameter to `generate_regular_rect_mesh` was actually a number of points per dimension; this change makes my suggested `nelements_per_axis` version equivalent to your original code.)